e2e/tests/smoke_test.go

```diff
@@ -26,7 +26,7 @@ const (
 	Timeout  = time.Second * 600
 	Interval = time.Second * 5
 
-	ydbImage      = "cr.yandex/crptqonuodf51kdj7a7d/ydb:22.4.44"
+	ydbImage      = "cr.yandex/crptqonuodf51kdj7a7d/ydb:22.5.10"
```

Comment:
@bma13 I'm not getting it to work. YDB doesn't really have 22.5.10 tag in an open source registry yet :( 

What we can do is:
- either leave 22.4.44 as it is
- or publish one of the newer builds into the oss registry

if we publish the newer build, I think we should also change the images in ./samples folder.